enhancements/bare-metal-fulfillment/README.md

```diff
@@ -0,0 +1,267 @@
+---
+title: bare-metal-fulfillment
+authors:
+  - Tzu-Mainn Chen
+creation-date: 2025-09-08
+last-updated: 2025-09-08
+tracking-link:
+  - None
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Bare Metal Fulfillment
+
+## Summary
+
+Bare metal fulfillment refers to the process through which a tenant acquires bare metal hosts and configures their networking.
```

Comment:
not sure if we want to integrate higher level use cases:
- Some tenants will want to deploy alternative clusters, e.g., SLURM
- We want to support broad use by Red Hat that involves developers installing their own cluster software, own OSes...